h/groups/views.py

```diff
@@ -91,7 +94,35 @@ def _read_group(request, group):
 
     """
     url = request.route_url('group_read', hashid=group.hashid, slug=group.slug)
-    template_data = {'group': group, 'group_url': url}
+
+    # TODO: Get shared annotations only.
+    result = search.search(request, params={
+        "group": group.hashid, "limit": 1000})
+    annotations = result['rows']
+
+    # Group the annotations by URI.
+    annotations_by_uri = collections.OrderedDict()
+    for annotation in annotations:
+        if annotation.uri in annotations_by_uri:
```

Comment:
I was thinking that was out of scope for this version, and that this whole thing will work much better once we have annotations and documents in a proper database, but I could probably just pass each uri through our `normalize()` function though